pkgs/by-name/cp/cppcheck/package.nix

```
@@ -73,13 +82,16 @@ stdenv.mkDerivation (finalAttrs: {
 
   meta = {
     description = "Static analysis tool for C/C++ code";
-    homepage = "http://cppcheck.sourceforge.net";
-    license = lib.licenses.gpl3Plus;
     longDescription = ''
       Check C/C++ code for memory leaks, mismatching allocation-deallocation,
       buffer overruns and more.
     '';
-    maintainers = with lib.maintainers; [ joachifm paveloom ];
+    homepage = "http://cppcheck.sourceforge.net";
+    license = lib.licenses.gpl3Plus;
     platforms = lib.platforms.unix;
+    maintainers = with lib.maintainers; [
+      joachifm
+      paveloom
+    ];
   };
```

```suggestion
    platforms = lib.platforms.unix;
  };
```

ASCIIbetical as before

---

My bad here, but I think the order from the [manual](https://nixos.org/manual/nixpkgs/unstable/#chap-meta) is more appropriate rather that ASCIIbetical (otherwise, I would need to put `longDescription` under `license`).

---

When I said "as before", I didn't mean it was ASCIIbetical, but that it was "more ASCIIbetical" before. My bad for not being clear.

---

P.S.: I prefer homepage first, because I believe it is the most important metas are, in order, homepage, description and longDescription.